src/LodeRunner.API/src/Controllers/LoadTestConfigsController.cs

```diff
@@ -91,11 +101,15 @@ public async Task<ActionResult> CreateLoadTestConfig([FromBody, SwaggerRequestBo
 
             if (newloadTestConfig.Validate(out string errorMessage, loadTestConfigPayload.PropertiesChanged))
             {
-                var insertedLoadTestConfig = await loadTestConfigService.Post(newloadTestConfig, cancellationTokenSource.Token);
+                var insertedLoadTestConfigResponse = await loadTestConfigService.Post(newloadTestConfig, cancellationTokenSource.Token);
 
-                if (insertedLoadTestConfig != null)
+                if (insertedLoadTestConfigResponse.Model != null && insertedLoadTestConfigResponse.StatusCode == HttpStatusCode.OK)
                 {
-                    return await ResultHandler.CreateResult(insertedLoadTestConfig, HttpStatusCode.OK);
+                    return await ResultHandler.CreateResult(insertedLoadTestConfigResponse.Model, HttpStatusCode.OK);
+                }
```

Comment:
We should return a 201 on create.

Reply:
Updated Returned status code in loadtestconfigs